xcodebuild/utils.go

```diff
@@ -147,11 +146,17 @@ func (b *xcodebuild) runPrettyXcodebuildCmd(useStdOut bool, xcprettyArgs []strin
 }
 
 func (b *xcodebuild) createXcodebuildTestArgs(params TestParams) ([]string, error) {
-	projectFlag := "-project"
-	if filepath.Ext(params.ProjectPath) == ".xcworkspace" {
-		projectFlag = "-workspace"
+	xcodebuildArgs := []string{}
+
+	fileExtension := filepath.Ext(params.ProjectPath)
+	if fileExtension == ".xcodeproj" {
+		xcodebuildArgs = append(xcodebuildArgs, "-project", params.ProjectPath)
+	}
+	if fileExtension == ".xcworkspace" {
+		xcodebuildArgs = append(xcodebuildArgs, "-workspace", params.ProjectPath)
```

Comment:
nitpick: There will be only one extension which will be either `xcodeproj` or `xcworkspace`. I had to double check it because this code piece hinted me that both can appear at the same time. I would change this to 
```
if fileExtension == ".xcodeproj" {
    xcodebuildArgs = append(xcodebuildArgs, "-project", params.ProjectPath)
} else if fileExtension == ".xcworkspace" {
    xcodebuildArgs = append(xcodebuildArgs, "-workspace", params.ProjectPath)
}
```

Reply:
Fixed, thank you for spotting this!